ingestion/src/metadata/ingestion/source/tableau.py

```diff
@@ -115,31 +99,39 @@ def tableau_client(self):
         Returns:
         """
         tableau_server_config = {
-            f"{self.config.env}": {
-                "server": self.config.server,
-                "api_version": self.config.api_version,
-                "site_name": self.config.site_name,
-                "site_url": self.config.site_url,
+            f"{self.config.serviceConnection.__root__.config.env}": {
```

Comment:
just aesthetic, but we are using `self.config.serviceConnection.__root__.config` a lot. We could prepare a variable `connection_config` during init to make the code more readable

Reply:
+1,
@OnkarVO7 can you take care the above

Reply:
okay I'll add a variable